test/framework/resources/templates/logger-app.yaml.tpl

```diff
@@ -51,7 +51,20 @@ spec:
 {{ else }}
           value: 0s
 {{- end }}
+        resources:
+          limits:
+            cpu: 8m
+            memory: 20Mi
+          requests:
+            cpu: 4m
+            memory: 3Mi
       securityContext:
         fsGroup: 65532
         runAsUser: 65532
         runAsNonRoot: true
+{{ if .NodeName }}
+      nodeName: {{ .NodeName }}
+{{- end }}
```

Comment:
What if the corresponding Node is already full and cannot run any logger Pod? What happens when the corresponding Node is not Ready? Are we sure that specifying nodeName (and hence skipping the kube-scheduler) we won't introduce any flake in the test?

Reply:
Yes, there is such a possibility but I don't want to rely on `topologySpreadConstraints` when we have to deal with only two nodes. The test is serial and there are enough resources.